app/web/assets/scss/patterns/_progress_bar.scss

```diff
@@ -0,0 +1,98 @@
+/* ==========================================================================
+   #PROGRESS BAR
+   ========================================================================== */
+
+.app-progress-bar {
+  margin-bottom: govuk-spacing(7);
+}
+
+.app-progress-bar__list {
+  font-size: 0; // Stop the connecting line from extending past the last item
+  list-style: none;
+  padding: 0;
+  position: relative;
+
+  // IE8 does not support the text justify approach for spacing
+  @include govuk-if-ie8 {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+  }
```

Comment:
This and other IE8 workarounds are straight from the existing horizontal component CSS and haven't been tested. Thinking maybe to drop them out and this can be revisited if needed when we/if we pollinate back to the pattern library?

Reply:
I'm fine with us doing a straight copy paste of all this stuff